### PR TITLE
small updates to the refbased rnaseq tutorial

### PR DESCRIPTION
**Pull request by shiltemann**
based on feedback/observations from teaching this recently. 

main changes: 
- clarify the downsampling and IGV steps
- the gtf file gets sniffed as gff by Galaxy and this will not work for the htseq-count tool

The Mac users also got a security warning from their browser when clicking the IGV links, blocking them from opening them, so if anybody has information about this that would be helpful to add here please let me know :)

### CODE REVIEW

**Comment by shiltemann** [comment]
also, this tutorial refers to the DAVID tool, which is not installable from any toolshed currently. @mvdbeek and others: which alternative tool would you recommend here?

**Comment by nsoranzo** [comment]
> also, this tutorial refers to the DAVID tool, which is not installable from any toolshed currently. @mvdbeek and others: which alternative tool would you recommend here?

xref. https://github.com/galaxyproject/training-material/issues/531

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 1]
```diff
@@ -182,17 +183,28 @@ We can now try to determine the library type of our data.
...
+>    > 2. How many rows must be selected to conserve 200k reads?
 >    >
 >    > <details>
 >    > <summary>Click to view answers</summary>
->    > In a FastQ file, a read corresponds to 4 lines. So to conserve 200,000 reads, 800,000 must be selected.
+>    > <ol>
+>    > <li> We are performing a preliminary mapping for the purpose of determining the library type. We do not need the full dataset for this, and
+>    >      mapping the full dataset can take quite some time and resources, so it is better to perform this test on a small subset of the data.
+>    > </li>
+>    > <li> In a FastQ file, a read corresponds to 4 lines. So to conserve 200,000 reads, 800,000 must be selected.
```
Comment (nsoranzo) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]: s/FastQ/FASTQ/

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 2]
```diff
@@ -182,17 +183,28 @@ We can now try to determine the library type of our data.
...
+>    >      More details about the fastq format can be found <a href="https://en.wikipedia.org/wiki/FASTQ_format">here</a>.
```
Comment (nsoranzo) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]: s/fastq/FASTQ/

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 3]
```diff
@@ -182,17 +183,28 @@ We can now try to determine the library type of our data.
...
+>    > </li>
+>    > </ol>
```
Comment (nsoranzo) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]: Is the HTML formatting necessary?

Comment (shiltemann) [reply topics/transcriptomics/tutorials/ref-based/tutorial.md]: it's what was done in the other question sections and it was either this or add some `<br>` to get the second answer to start on a new line

**Comment by bebatut** [comment]
I merge it now. @shiltemann will replace DAVID later

**Pull request merged**